test/unit/lib/plugins/aws/package/compile/events/cloud-front.test.js

```diff
@@ -1715,33 +1454,122 @@ describe('test/unit/lib/plugins/aws/package/compile/events/cloudFront.test.js',
     });
   });
 
-  describe.skip('TODO: Alternative cases', () => {
+  describe('Alternative cases', () => {
     it('should not create cloudfront distribution when no cloudFront events are given', async () => {
-      // Replaces
-      // https://github.com/serverless/serverless/blob/85e480b5771d5deeb45ae5eb586723c26cf61a90/lib/plugins/aws/package/compile/events/cloudFront/index.test.js#L572-L593
-
-      await runServerless({ fixture: 'function', command: 'package' });
+      return expect(
+        await runServerless({
+          fixture: 'function',
+          command: 'package',
+          configExt: {
+            functions: {
+              first: {
+                handler: 'first.handler',
+                name: 'first',
+              },
+              second: {
+                name: 'second',
+                handler: 'second.handler',
+                events: [
+                  {
+                    http: 'GET /',
+                  },
+                ],
+              },
+            },
+          },
+        })
+      ).to.not.have.any.keys('CloudFrontDistribution');
     });
 
-    it('should create DefaultCacheBehavior if there are no events without PathPattern configured', async () => {
+    it.skip('TODO: should create DefaultCacheBehavior if there are no events without PathPattern configured', async () => {
```

Comment:
Is there a specific reason why this one is not addressed?

Reply:
as mentioned here i am getting a little weird error 
https://github.com/serverless/serverless/issues/8528#issuecomment-1383959217

Reply:
Ah yes, it's because `10240` is in general maximum allowed memory size. In such case I believe we can remove such validation from event implementation and also remove test for it. As restriction doesn't lie on CF event side

Reply:
Sorry wrong thread used for another conversation. This one i was not able to get what we are trying to do.

Reply:
completed that as well